README.md

```diff
@@ -1 +1,85 @@
-# Awesome-Book-ES6
+
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <img src="/murple_logo.png" alt="logo" width="140"  height="auto" />
```

Comment:
- [ ] Kindly replace this logo with your own logo or get rid of it. This project is meant for you alone.